### PR TITLE
gridtracker2 2.241107.2 (renamed from gridtracker)

### PR DESCRIPTION
**Pull request by daeho-ro**
Major update

---

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/g/gridtracker.rb [hunk 1]
```diff
@@ -1,46 +1,24 @@
...
-  url "https://downloads.gridtracker.org/mac_v#{version.csv.first}/GridTracker-#{version.csv.first}-mac-#{version.csv.second}-x64.zip"
+  url "https://download2.gridtracker.org/GridTracker2-#{version}-universal-mac.zip"
   name "GridTracker"
   desc "Warehouse of amateur radio information presented in an easy to use interface"
   homepage "https://gridtracker.org/"
 
-  depends_on macos: ">= :mojave"
+  livecheck do
+    url "https://gridtracker.org/index.php/downloads/gridtracker-downloads"
+    regex(/href=.*?GridTracker2[._-]v?(\d+(?:\.\d+)+)[._-]universal-mac\.zip/i)
```
Comment (khipp) [inline Casks/g/gridtracker.rb:12]:
```suggestion
    url "https://download2.gridtracker.org/latest-mac.yml"
    strategy :electron_builder
```

**Comment by khipp** [comment]
It seems like the app is now called `GridTracker2` (see [upstream repository)](https://gitlab.com/gridtracker.org/gridtracker2). We should probably update the cask's `token` and `name`.

**Comment by krehel** [comment]
I don't think there's harm in keeping the previous zap entries to ensure cleanup.

File: Casks/g/gridtracker2.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+cask "gridtracker2" do
+  version "2.241107.2"
+  sha256 "abd3b79192b109f7df2947c36d2ed5f6c4e258c85a7f44592f4c0df0d4ae5f73"
+
+  url "https://download2.gridtracker.org/GridTracker2-#{version}-universal-mac.zip"
+  name "GridTracker"
```
Comment (khipp) [inline Casks/g/gridtracker2.rb:6]:
```suggestion
  name "GridTracker2"
```

File: Casks/g/gridtracker2.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  app "GridTracker2.app"
+
+  zap trash: [
+    "~/Library/Application Support/GridTracker",
+    "~/Library/Application Support/GridTracker2",
+    "~/Library/Caches/GridTracker",
+    "~/Library/Preferences/org.gridtracker.gridtracker.plist",
+    "~/Library/Preferences/org.gridtracker.GridTracker2.plist",
+    "~/Library/Saved Application State/org.gridtracker.gridtracker.savedState",
+    "~/Library/Saved Application State/org.gridtracker.GridTracker2.savedState",
```
Comment (khipp) [inline Casks/g/gridtracker2.rb:26]:
```suggestion
    "~/Library/Application Support/GridTracker*",
    "~/Library/Caches/GridTracker",
    "~/Library/Preferences/org.gridtracker.GridTracker*.plist",
    "~/Library/Saved Application State/org.gridtracker.GridTracker*.savedState",
```

We can combine these entries using a glob `*`.

Comment (daeho-ro) [reply Casks/g/gridtracker2.rb:26]: This is cool!

**Review by khipp — APPROVED** [review]

**Pull request merged**